yt_dlp/extractor/vvvvid.py

```diff
@@ -78,11 +113,24 @@ class VVVVIDIE(InfoExtractor):
         'only_matching': True
     }]
     _conn_id = None
+    _default_user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.50 Safari/537.37'
+    _blocked_user_agents_regex = r'^Mozilla/5\.0 \(Windows NT 10\.0; Win64; x64\) AppleWebKit/537\.36 \(KHTML, like Gecko\) Chrome/[\d.]+ Safari/537\.36$'
+    _headers = {}
+
+    def _get_headers(self):
+        geo_verification_headers = self.geo_verification_headers()
+        http_headers = self.get_param('http_headers', {})
+        headers = {**http_headers, **geo_verification_headers}
+        user_agent = headers.get('User-Agent', None)
+        if not user_agent or re.match(self._blocked_user_agents_regex, user_agent):
+            headers['User-Agent'] = self._default_user_agent
+        return headers
```

Comment:
Just add the custom user agent unconditionally imo. It's unlikely anyone wants custom UA here unless IE breaks.

Also, `_headers` looks like it should be a `@cached_property`